### PR TITLE
Resolves #1351: Add labeling guide to front page

### LINKED ISSUE #1351: Labeling guide should be visible on landing page front menu
The labeling guide should be a top-level link on the landing page imo.

image

### PR DESCRIPTION
**Pull request by aileenzeng**
Adds labeling guide to the front page (navbar). 
Before:
image

After:
image

I also modified the Labeling Guide description on the audit screen for consistency:

Before:
image

After: 
image

### CODE REVIEW

File: app/views/navbar.scala.html [hunk 1]
```diff
@@ -17,6 +17,9 @@
                    <li class="active navbarLink">
                         <a class="navbarBtn navbarStartBtn" href="@routes.AuditController.audit()">Start Mapping</a>
                     </li>
+                    <li class="active navbarLink">
+                        <a class="navbarBtn navbarStartBtn" href="@routes.ApplicationController.labelingGuide">How to Label</a>
```
[the remark this answers is not in the document]
Comment (aileenzeng) [reply app/views/navbar.scala.html]: Thanks! This should be fixed now.

**Comment by aileenzeng** [comment]
Main page navbar:
image
(Links: `Start Mapping`, `How to Label`, `See Results`)

Audit navbar (what we have currently):
image
(Links: `Retake tutorial`, `How to label`, `Help`)

@misaugstad and I were talking about this earlier - there seem to be some inconsistencies in how we capitalize links in the navbar. Should we change the audit navbar so that the beginning of each word is capitalized to match the style of our main navbar? 

image
(Links: `Retake Tutorial`, `How to Label`, `Help`)

**Comment by misaugstad** [comment]
@jonfroehlich looking for your input here. I vote to add the capitalization to the audit page (i.e., Retaking tutorial -> Retake Tutorial and How to label -> How to Label).

**Comment by aileenzeng** [comment]
I agree with @misaugstad!

**Comment by jonfroehlich** [comment]
Sounds fine

**Comment by misaugstad** [comment]
@aileenzeng when updating the capitalization, make sure to update it in the logging portion as well

**Comment by aileenzeng** [comment]
Should be ready now!

**Comment by misaugstad** [comment]
Awesome, thanks! Looks good!

**Pull request merged**